resources/assets/js/validations.js

```diff
@@ -376,7 +376,10 @@ $.extend(true, laravelValidation, {
          * @return {boolean}
          */
         Mimes: function(value, element, params) {
-            var lowerParams = $.map(params, String.toLowerCase);
+            var lowerParams = $.map(params, function(item, index) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["index" is defined but never used](https://www.codacy.com/app/albert-moreno-t/laravel-jsvalidation/file/2183766929/issues/source?bid=2225045&fileBranchId=3117047#l379)